packages/sdk-bundles-api/src/replay/sdk-to-bundle/index.ts

```diff
@@ -8,8 +8,7 @@ export interface ReplayUserInteractionsOptions {
   page: Page;
   sessionData: unknown;
   moveBeforeClick: boolean;
-  acceleratePlayback: boolean;
-  virtualTime: VirtualTimeOptions;
+  virtualTime?: VirtualTimeOptions;
```

Comment:
Since this was an added field it needs to be optional to ensure back compat when consuming these types from the bundle side.